roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -175,6 +176,11 @@ server {
     fastcgi_cache_bypass $skip_cache;
     fastcgi_no_cache $skip_cache;
 
+    # Update an expired cache item in the background, serve stale cache to client.
+    {% if item.value.cache.background_update | default(true) %}
+    fastcgi_cache_background_update on;
```

Comment:
Sorry for the confusion I believe you should remove the if and simply put the one-liner
`fastcgi_cache_background_update {{ item.value.cache.background_update | default(on) }};`
Also, please make sure to test if you have not done so :)

Thank you for being patient with us and thank you for your contribution :)